### ISSUE TITLE
Cert Hostname DOES NOT VERIFY (* != simple-mail-forwarder.com)

### ISSUE THREAD

**Issue by Mygod** [huan/docker-simple-mail-forwarder#71]
Is there any way to change this? Gmail has started to refuse to work with this since earlier this month. checktls.com also catches this error.

Related thread: https://support.google.com/mail/thread/38336515?msgid=39890656

I tried setting `-e SMF_DOMAIN=mydomain.com` as #10 but no luck.

**Comment by JeffHiggy** [comment]
Does SMF_DOMAIN, your public DNS record, and the common name the certificate is issued to all match?

**Comment by Mygod** [comment]
Yes. I noted that the Dockerfile sets /etc/hostname to this and it's not configurable. Maybe that's the issue?

**Comment by petslane** [comment]
Probably this is your problem https://github.com/huan/docker-simple-mail-forwarder/pull/51#issuecomment-688402018

**Comment by Mygod** [comment]
Yes.

**Comment by zeroleo12345** [comment]
+1. stupid documentation
